<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter_25">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>361</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter_24">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_22">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_21">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Dobot не подключен</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QSplitter" name="splitter_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="splitter_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Текущие координаты:</string>
             </property>
            </widget>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>X:</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="currentBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="minimum">
               <number>-999</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="currentBox2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="minimum">
               <number>-999</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Z:</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="currentBox3">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="minimum">
               <number>-999</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </widget>
            <widget class="QPushButton" name="pushButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Обновить</string>
             </property>
            </widget>
           </widget>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
           <widget class="QSplitter" name="splitter_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Введите координаты:</string>
             </property>
            </widget>
            <widget class="QSplitter" name="splitter_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>X:</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="setBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="minimum">
               <number>-999</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="setBox2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="minimum">
               <number>-999</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Z:</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="setBox3">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="minimum">
               <number>-999</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </widget>
            <widget class="QPushButton" name="pushButton_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Переместить</string>
             </property>
            </widget>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QPushButton" name="connectButton">
         <property name="text">
          <string>Подключить Dobot</string>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_20">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Движение рельсы</string>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSpinBox" name="railBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
          <widget class="QPushButton" name="railMove">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Переместить</string>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Движение конвейера</string>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter_19">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSpinBox" name="convBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="minimum">
            <number>-10000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
          <widget class="QPushButton" name="convMove">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Переместить</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="homeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Домой</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_23">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="socButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Включить присоску</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gripButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Включить захват</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
